common/history/countries/aus - austria.txt

```diff
@@ -3,6 +3,36 @@
 		# Politics
 		#todo IGs and Ideologies
 		# Tech and Tax
+		set_tax_level = high
+		effect_create_law_package = {
+			headOfGovernment = council_elective
```

Comment:
soviet occupied austria should be 
administrative as it's a military occupation.